### ISSUE TITLE
Changes export = to export default when formatting typescript.

### ISSUE THREAD

**Issue by frederickfogerty** [prettier/prettier#1860]
Hey there. Thanks for the awesome lib 🙌🏿

Prettier converts this:
```ts
declare module 'react-tagsinput' {
	export = "Something"
}
```
to
```ts
declare module "react-tagsinput" {
  export default "Something";
}
```

Which is semantically different, and causes my build to fail.

https://prettier.github.io/prettier/#%7B%22content%22%3A%22declare%20module%20'react-tagsinput'%20%7B%5Cn%5Ctexport%20%3D%20%5C%22Something%5C%22%5Cn%7D%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22typescript%22%2C%22semi%22%3Atrue%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%7D%7D

**Comment by frederickfogerty** [comment]
Closing as dupe of #1853

**Comment by vjeux** [comment]
I just published 1.4.1 which fixes it!
